packages/core/src/components/header/header-item/header-item.tsx

```diff
@@ -51,10 +53,46 @@ export class TdsHeaderItem {
     }
   }
 
+  /**
+   * Read the native "order" attribute on the host and update the CSS order accordingly.
+   * If the attribute "order" has the value "end", the order is set to 1. Otherwise, 0.
+   */
+  updateOrder() {
```

Comment:
Maybe this is a good place to use switch :) 
```suggestion
  updateOrder() {
    const orderValue = this.host.getAttribute('order');
    switch (orderValue) {
      case 'end':
        this.host.style.order = '1';
        break;
      case 'start':
      case null:
      default:
        this.host.style.order = '0';
        break;
    }
  }
```

Reply:
Sure! 😄